theme/templates/includes/access-row.html

```diff
@@ -1,133 +1,89 @@
 {% load hydroshare_tags %}
 
-{% for u in entries %}
-<tr id="row-id-{{ u.pk }}"
-    {% if access != "Is owner" %}
-    {% elif entries|length == 1 or not is_owner_user or u.pk == quota_holder.pk %}
-    class="hide-actions"
-    {% endif %}>
+<tr v-for="(user, index) in this.users"
+    :class="{ 'hide-actions': selfAccessLevel !== 'owner' && (user.access == 'owner' && hasOnlyOneOwner || user.id === quotaHolder.id), loading: user.loading}">
     <td>
```

Comment:
I still get confused when to use double or triple = between user.access and 'owner' do you mean for three?

Reply:
I do. I will make this change. Generally is good to use '===' because it will also check for the type.